test/e2e/util/hypershift_framework.go

```diff
@@ -197,6 +197,8 @@ func (h *hypershiftTest) after(hostedCluster *hyperv1.HostedCluster, platform hy
 
 		ValidateMetrics(t, context.Background(), h.client, hostedCluster, metricsToValidate, true)
 
+		numNodes := opts.NodePoolReplicas * int32(len(opts.AWSPlatform.Zones))
+		ValidateHostedClusterConditions(t, context.Background(), h.client, hostedCluster, numNodes > 0, 10*time.Minute)
```

Comment:
we should really not be using `context.Background()` and use the test context `h.ctx`
I know this is already used on all the other tests in `after()`, I am not sure why